tests/lib/Controller/AggregatorTest.php

```diff
@@ -0,0 +1,168 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimpleSAML\Test\Module\aggregator\Controller;
+
+use PHPUnit\Framework\TestCase;
+use SimpleSAML\Configuration;
+use SimpleSAML\Error;
+use SimpleSAML\Module\aggregator2\Controller;
+use SimpleSAML\Session;
+use SimpleSAML\XHTML\Template;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+//use function call_user_func;
+
+/**
+ * Set of tests for the controllers in the "aggregator2" module.
+ *
+ * @covers \SimpleSAML\Module\aggregator2\Controller\Aggregator
+ */
+class AutotestTest extends TestCase
```

Comment:
Wrong class name?

Reply:
Yeah, copy/paste mistake! Thanks for catching this?
Are you by any chance familiar with this module, because I'm not and haven't really tested this yet

Reply:
Only passingly